Dockerfile

```
@@ -1,5 +1,5 @@
-FROM python:3.13
-
-ADD App/requirements.txt ./
-
-RUN pip install -r requirements.txt
\ No newline at end of file
+FROM python:3.13
+
+ADD app/requirements.txt ./
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)